Dockerfiles/agent/README.md

```diff
@@ -27,6 +27,17 @@ The agent is highly customizable, here are the most used environment variables:
 - `DD_TAGS`: host tags, separated by spaces. For example: `simple-tag-0 tag-key-1:tag-value-1`
 - `DD_CHECK_RUNNERS`: the agent runs all checks in sequence by default (default value = `1` runner). If you need to run a high number of checks (or slow checks) the `collector-queue` component might fall behind and fail the healthcheck. You can increase the number of runners to run checks in parallel
 
+#### Proxies
+
+The agent proxy settings can be overridden with the standard `*_PROXY`
+environment variables:
+
+- `HTTP_PROXY`: an http URL to use as a proxy for `http` requests.
+- `HTTS_PROXY`: an http URL to use as a proxy for `https` requests.
+- `NO_PROXY`: a coma-separated list of URLs for which no proxy should be used.
+
+Notice: these variables don't use the `DD_` prefix.
+
```

Comment:
we could also maybe link to the proxy docs on the docs website (https://docs.datadoghq.com/agent/proxy/#agent-v6) from here? It'd be good to centralize as much proxy-related docs/questions as possible on the docs website so that we have this one reference we an easily link to.